root_data.tf

```diff
@@ -69,3 +69,15 @@ data "aws_ssm_parameter" "slack_success_workflow" {
 data "aws_ssm_parameter" "slack_failure_workflow" {
   name = "/mgmt/slack_failure_workflow"
 }
+
+data "aws_ssm_parameter" "enterprise_gpg_key_id" {
+  name = "/mgmt/github_enterprise/gpg/key_id"
+}
+
+data "aws_ssm_parameter" "enterprise_gpg_private_key" {
+  name = "/mgmt/github_enterprise/gpg/private_key"
+}
+
+data "aws_ssm_parameter" "enterprise_gpg_passphrase" {
+  name = "/mgmt/github_enterprise/gpg/passphrase"
```

Comment:
Do we need to declare the same parameter again if we already declared it in the `root.tf`?

Reply:
> _Do we need to declare the same parameter again if we already declared it in the root.tf?_

@vimleshtna Strictly speaking no. But this is tidier for other modules to use, ie `data.aws_ssm_parameter.enterprise_gpg_passphrase.value` as opposed to doing this `local.github_enterprise_gpg_passphrase_parameter.value`. It not relying on the name of the local, ie `..._parameter` to define what it is, as its clear due to the `...aws_ssm_parameter...`